changelog.md

```diff
@@ -8,6 +8,13 @@ class: changelog
 
 {% capture changelog %}
 
+## Version 5.0.15 September 2, 2019
+* Added a dark content_css styles to go with the dark skin.
```

Comment:
Added the dark content_css styles....?

Reply:
I think `a` is right here as it's not the one and only value, but maybe it should just be `style` instead of `styles` or even potentially `content_css skin`?

Reply:
sounds good. had a feeling it was one of those :)